WrightTools/artists.py

```diff
@@ -103,9 +106,20 @@ def _apply_labels(self, autolabel='none', xlabel=None, ylabel=None, data=None, c
             self.set_ylabel(ylabel, fontsize=18)
 
     def _parse_limits(self, zi=None, data=None, channel_index=None, dynamic_range=False, **kwargs):
+        null = kwargs.get('null', 0)
         if zi is not None:
-            vmin = np.nanmin(zi)
-            vmax = np.nanmax(zi)
+            if 'levels' in kwargs.keys():
+                levels = kwargs['levels']
+                vmin = levels.min()
+                vmax = levels.max()
+            elif 'signed' in kwargs.keys():
+                if kwargs['signed'] and not dynamic_range:
+                    mag = max(np.abs([np.nanmin(zi-null), np.nanmax(zi-null)]))
+                    vmin = null - mag
+                    vmax = null + mag
```

Comment:
What about the case where dynamic range is True?

Reply:
Again, removed because `signed` kwarg is not desired (346b527f620f45109ccd3d0035e016ecdfbf0f06)